pennylane/transforms/insert_ops.py

```diff
@@ -102,7 +101,7 @@ def insert(
 
         dev = qml.device("default.mixed", wires=2)
 
-        @partial(qml.transforms.insert, qml.AmplitudeDamping, 0.2, position="end")
```

Comment:
Maybe we should stick a `from functools import partial` at the top of this codeblock, in case users are confused where `partial` is coming from.